smartmin/views.py

```diff
@@ -97,34 +95,7 @@ def has_permission(self, request, *args, **kwargs):
         if not getattr(self, 'permission', None):
             return True
         else:
-            # first check our anonymous permissions
-            real_anon = get_anonymous_user()
-            has_perm = real_anon.has_perm(self.permission)
```

Comment:
I guess that's an ok compromise for removing guardian.. can add something else in that is fancier if we ever want to return to that being a configuration level attribute.